src/modules/fancyzones/lib/Resources.resx

```diff
@@ -213,4 +213,7 @@
   <data name="Powertoys_FancyZones" xml:space="preserve">
     <value>PowerToys - FancyZones</value>
   </data>
+  <data name="Span_Across_Zones_Warning" xml:space="preserve">
+    <value>With 'Allow zones to span across monitors' setting ON, all monitors must have the same DPI scaling</value>
```

Comment:
With 'Allow zones to span across monitors' setting ON, all monitors must have the same DPI scaling 
->
The 'Allow zones to span across monitors' option requires all monitors to have the same DPI scaling to work properly.